app/src/main/kotlin/com/mapzen/erasermap/util/DisplayHelper.kt

```diff
@@ -3,15 +3,33 @@ package com.mapzen.erasermap.util
 import android.content.Context
 
 public object DisplayHelper {
+    @JvmStatic private val ROUTE_ICON_PREFIX = "ic_route_"
+    @JvmStatic private val ROUTE_ICON_POSTFIX_LARGE = "_lg"
+    @JvmStatic private val DRAWABLE_PREFIX = "drawable"
 
     /**
      * Fetch the resource drawable ID of the turn icon for the given instruction and style.
      * @param context current context in which to display the icon.
-     * @param turnInstruction the integer value representing this turn instruction.
+     * @param type the integer value representing this maneuver type.
      * @return the resource ID of the turn icon to display.
      */
-    public fun getRouteDrawable(context: Context, turnInstruction: Int?): Int {
-        return context.getResources().getIdentifier("ic_route_"
-                + turnInstruction, "drawable", context.getPackageName())
+    public fun getRouteDrawable(context: Context, type: Int?): Int {
+        return getRouteDrawableInternal(context, type, false)
+    }
+
+    /**
+     * Fetch the resource drawable ID of the large turn icon for the given instruction and style.
+     * @param context current context in which to display the icon.
+     * @param type the integer value representing this maneuver type.
+     * @return the resource ID of the turn icon to display.
+     */
+    public fun getRouteDrawableLarge(context: Context, type: Int?): Int {
+        return getRouteDrawableInternal(context, type, true)
+    }
+
+    private fun getRouteDrawableInternal(context: Context, type: Int?, isLarge: Boolean): Int {
```

Comment:
What's your reasoning for having separate functions for different size route drawables and not just have the code call the boolean-based function directly?